[libcxx] Use %{temp} instead of %T

boomanaiden154 opened the pull request:
Based on review feedback in #160026.

This makes the substitution a lot more clear now that there is no documentation around %T.

boomanaiden154 committed:
[libcxx] Use %{temp} instead of %T

Based on review feedback in #160026.

This makes the substitution a lot more clear now that there is no
documentation around %T.

boomanaiden154 committed:
formatting changes

boomanaiden154 committed:
fix cxxabi/unwind

ldionne reviewed (approved):
Thanks for the patch! LGTM but can you please undo the formatting changes that were done in this patch?

boomanaiden154 committed:
Revert "formatting changes"

This reverts commit 6d5ae7ece7e970f40d008f6e6ee84d3206e1c7fd.

ldionne committed:
Move the documentation of %{temp} to where other substitutions are documented

We should document everything in the same place, but for now let's be
consistent.

ldionne reviewed (approved):
LGTM, thanks!
